packages/logger/package.json

```diff
@@ -19,8 +19,6 @@
   },
   "scripts": {
     "build": "cross-env NODE_ENV=production tsc",
-    "test": "cross-env mocha --exit -r ../../babel-register.js --recursive --opts ./test/mocha.opts ./test/unit",
-    "test-debug": "cross-env mocha --inspect-brk --exit -r ../../babel-register.js --recursive --opts ./test/mocha.opts ./test/unit",
```

Comment:
this package and commons really didn't have tests, so i didn't replace the script commands until we're ready to add tests to those packages.  The functionally and code off logger and commons are tested numerous times by the other package integration tests.